packages/cli/src/workflows/workflows.services.ts

```diff
@@ -463,7 +464,13 @@ export class WorkflowsService {
 			await Container.get(ActiveWorkflowRunner).remove(workflowId);
 		}
 
+		const idsForDeletion = await Db.collections.Execution.find({
+			select: ['id'],
+			where: { workflowId },
+		}).then((rows) => rows.map(({ id: executionId }) => ({ workflowId, executionId })));
+
 		await Db.collections.Workflow.delete(workflowId);
+		await Container.get(BinaryDataService).deleteMany(idsForDeletion);
```

Comment:
I'm slightly worried about this potentially timing out for a user if there's a lot of files on a slow file system. I don't really think it's worth changing, just thought I'd put it out there.

Reply:
Please see cbb84c3e48354f4879ddf9016fc80b0e2af6ba00